qiskit/primitives/backend_estimator.py

```diff
@@ -104,6 +105,7 @@ class BackendEstimator(BaseEstimator[PrimitiveJob[EstimatorResult]]):
     precludes doing any provider- or backend-specific optimizations.
     """
 
+    @deprecate_func(since="1.0", additional_msg="Use BackendEstimatorV2 instead.")
```

Comment:
Thank you for working on this task.

It will be deprecated with the next major release. So, could you update all `since` values and deprecation messages?
```suggestion
    @deprecate_func(since="1.2", additional_msg="Use BackendEstimatorV2 instead.")
```